app/_hub/kong-inc/jwt/_index.md

```diff
@@ -529,52 +537,52 @@ You can paginate through the JWTs for all Consumers using the following
 request:
 
 ```bash
-curl -X GET http://kong:8001/jwts
+curl -X GET http://localhost:8001/jwts
 ```
 
 Response:
 ```json
 {
-    "total": 3,
-    "data": [
-        {
-            "created_at": 1509593911000,
-            "id": "381879e5-04a1-4c8a-9517-f85fbf90c3bc",
-            "algorithm": "HS256",
-            "key": "UHVwIly5ZxZH7g52E0HRlFkFC09v9yI0",
-            "secret": "KMWyDsTTcZgqqyOGgRWTDgZtIyWeEtJh",
-            "consumer": { "id": "3c2c8fc1-7245-4fbb-b48b-e5947e1ce941" }
-        },
-        {
-            "created_at": 1511389527000,
-            "id": "0dfc969b-02be-42ae-9d98-e04ed1c05850",
-            "algorithm": "ES256",
-            "key": "vcc1NlsPfK3N6uU03YdNrDZhzmFF4S19",
-            "secret": "b65Rs6wvnWPYaCEypNU7FnMOZ4lfMGM7",
-            "consumer": { "id": "c0d92ba9-8306-482a-b60d-0cfdd2f0e880" }
-        },
-        {
-            "created_at": 1509593912000,
-            "id": "d10c6f3b-71f1-424e-b1db-366abb783460",
-            "algorithm": "HS256",
-            "key": "SqSNfg9ARmPnpycyJSMAc2uR6nxdmc9S",
-            "secret": "CCh6ZIcwDSOIWacqkkWoJ0FWdZ5eTqrx",
-            "consumer": { "id": "3c2c8fc1-7245-4fbb-b48b-e5947e1ce941" }
-        }
-    ]
+	"next": null,
+	"data": [
+		{
+			"id": "0701ad83-949c-423f-b553-091d5a6bae52",
+			"secret": "C50k0bcahDhLNhLKSUBSR1OMiFGzNZ7X",
+			"key": "YJdmaDvVTJxtcWRCvkMikc8oELgAVNcz",
+			"tags": null,
+			"rsa_public_key": null,
+			"consumer": {
+				"id": "8a21c1fa-e65e-4558-8673-540e85e67b33"
+			},
+			"algorithm": "HS256",
+			"created_at": 1664462115
+		},
+		{
+			"id": "e14d775e-3b52-45cc-be9e-b39cdb3f7ebf",
+			"secret": "gFrOZAYWH1osQ6ivkesT4qqH16DHpKu0",
+			"key": "7TrSUEFcEP7KZb2QrGT9IQXcEWrmszC8",
+			"tags": null,
+			"rsa_public_key": null,
+			"consumer": {
+				"id": "8a21c1fa-e65e-4558-8673-540e85e67b33"
+			},
+			"algorithm": "HS256",
+			"created_at": 1664398496
+		}
+	]
 }
 ```
 
-You can filter the list by consumer by using this other path:
+You can filter the list by consumer by using another path:
 
 ```bash
-curl -X GET http://kong:8001/consumers/{username or id}/jwt
+curl -X GET http://localhost:8001/consumers/USERNAME|ID/jwt
 ```
 
 Response:
 ```json
 {
-    "total": 1,
+    "next": null,
     "data": [
         {
             "created_at": 1511389527000,
```

Comment:
@Guaris I think the "consumer" value below this still has the old formatting

Reply:
nice catch. I found another one too. Added that in my latest commit.